rad/src/error.rs

```diff
@@ -448,6 +447,7 @@ impl RadError {
                     message: Some(message),
                 }
             }
+            _ => RadError::Unknown,
```

Comment:
I would prefer to be explicit to avoid problems if we add more errors in the future and forget to change it here...

```suggestion
            RadonErrors::Unknown | RadonErrors::BridgeMalformedRequest | etc => RadError::Unknown,
```